whylog/teacher/__init__.py

```diff
@@ -0,0 +1,58 @@
+class Interval(object):
+    """
+    Represents interval in line.
+    """
+
+    def __init__(self, start_offset, length, abstract_line):
+        self.start_offset = start_offset
+        self.length = length
+        self.abstract_line = abstract_line
```

Comment:
`abstract_` prefix again, see note for `whylog/constraints/__init__.py` line `11`